packages/holocron/__tests__/reactStreaming.spec.jsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies -- monorepo, this is at the root
```

Comment:
Wouldn't it make more sense to include it in the devDependencies for this workspace than to disable the rule?